src/Modules/Render/ViewScene.h

```diff
@@ -158,6 +159,14 @@ namespace Render {
     void syncMeshComponentFlags(const std::string& connectedModuleId, Dataflow::Networks::ModuleStateHandle state);
 
     ActiveGeometryMap activeGeoms_;
+
+    class SCISHARE ScopedExecutionReporter
+    {
+      Dataflow::Networks::ModuleStateHandle state_;
+    public:
+      ScopedExecutionReporter(Dataflow::Networks::ModuleStateHandle state);
```

Comment:
Sorry missed these easy improvements earlier
```suggestion
      explicit ScopedExecutionReporter(Dataflow::Networks::ModuleStateHandle state);
```